### PR TITLE
Added clarity to README.md and expanded howto.

### PR DESCRIPTION
**Pull request by chamunks**
Added a section for Running from Release this may need improvement.
Removed some whitespace.
Linked to official GoLang installation guide in Prerequisites section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,10 @@
 
 Make sure you have Go 1.10+ (required for [go-releaser](https://goreleaser.com/))
 
-- Ubuntu: https://launchpad.net/~longsleep/+archive/ubuntu/golang-backports or https://github.com/golang/go/wiki/Ubuntu
-- OSX: `brew install go`
+- Ubuntu: https://launchpad.net/~longsleep/+archive/ubuntu/golang-backports or https://github.com/golang/go/wiki/Ubuntu [Easier but maybe older] **(recommended)**
+- Linux: [GoLang official install guide](https://golang.org/doc/install#tarball) [Harder but up to date] 
+- OSX: [GoLang official install guide](https://golang.org/doc/install#macos)
+- Windows: Not supported, there will be ways to run it on Windows if you use Docker.
```
Comment (tiger5226) [inline README.md]: I tested the Windows build and it works fine. Did you find out differently?

**Comment by chamunks** [comment]
@tiger5226 I was unaware that there was Windows support I'll cancel this PR and submit a new one with a few adjustments.

File: README.md (hunk 1 above)
Comment (chamunks) [reply README.md]: Is there documentation on how to compile it in go as I think this section is to explain how to do that.  This section didn't have the generic Linux section before so I added that.  Then I added the windows section while making assumptions that there wasn't a Windows build as it wasn't documented prior.  I mostly wanted to add official go docs for installing go not just what we use.

File: README.md [hunk 2]
```diff
@@ -64,6 +65,47 @@ go get -u github.com/lbryio/chainquery
...
+  ```
+Of course you don't have to extract all of this stuff to your machines home directory `~/` you must use whatever paths you prefer.  One that could be beneficial is adding these executables into your systems `$PATH` this is out of the scope of this README.
+
+The main Chainquery binary should be marked as Executable by default but if not you can run the following.
+```
+chmod +x ~/chainquery
+```
+Finally running chainquery should be as simple as.
+```
+~/chainquery
```
[the remark this answers is not in the document]
Comment (chamunks) [reply README.md]: Updated in next PR

**Pull request closed without merging**